src/components/ProfileForm/components/ProfileFormStep0.tsx

```diff
@@ -0,0 +1,59 @@
+const ProfileFormStep0 = () => {
+  return (
+    <>
+      <form className="m-1 p-1 mb-4">
+        <fieldset className="mb-2">
+          <legend className="text-base mb-2 font-bold">
+            Which gender do you identify as?
+          </legend>
+          <div className="flex flex-col">
+            <label>
+              <input name="gender" value="female" type="radio" />
+              <span className="pl-1"> Female </span>
```

Comment:
Looking at this component, and the others, there's a lot of duplicate code here, writing the `span`, making sure we add the correct `pl-1` class and so on.

Let's create a new `ProfileFormLabelledInput` component that encapsulates this logic, so we duplicate as little code as possible.

Reply:
An example use of the component is:

```
<ProfileFormLabelledInput label="Male">
  <input type="radio" name="gender" value="male" />
</ProfileFormLabelledInput>
```